syntax = "proto3";

package rbk4.protocol;

import "message_header.proto";


message Message_Odometer {
    Message_Header header = 1;
    uint32 cycle = 2;  // cnt
    double x = 3;  // m
    double y = 4;  // m
    float angle = 5;  // rad
    bool is_stop = 6;
    float vel_x = 7; // m/s
    float vel_y = 8; // m/s
    float vel_rotate = 9; // rad/s
}